src/main/java/com/vinylteam/vinyl/PropertiesReader.java

```diff
@@ -0,0 +1,55 @@
+package com.vinylteam.vinyl;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesReader {
+
+    public PropertiesReader() {
+        InputStream inputStream;
+
+        inputStream = getClass().getClassLoader().getResourceAsStream("application.properties");
+
+        setProperties(inputStream);
+    }
+
+    private final Properties properties = new Properties();
+
+    private void setProperties(InputStream inputStream) {
+        try {
+            properties.load(inputStream);
+        } catch (IOException e) {
+            throw new RuntimeException("Was unable to load properties", e);
```

Comment:
1. why we don't use try-with-resources? We should close inputStream
2. Why we create another method setProperties? We can load inputstream in PropertiesReader

Reply:
public class PropertyReader {

    Properties readProperties() {
        Properties properties = new Properties();
        try (InputStream inputStream = getClass().getClassLoader().getResourceAsStream("application.properties")) {
            if (inputStream == null){
                throw new RuntimeException("File 'application.properties' not found!");
            }
            properties.load(inputStream);
        } catch (IOException e) {
            throw new RuntimeException("Error during load properties!", e);
        }
        return properties;
    }
}

Reply:
Got it, thank you!